snis_client.c

```diff
@@ -20703,6 +20703,7 @@ static void connect_to_lobby_button_pressed(__attribute__((unused)) void *unused
 	rc = sscanf(net_setup_ui.lobbyportstr, "%d", &lobbyport);
 	if (rc != 1 || use_default_lobby_port)
 		lobbyport = -1; /* let ssgl use default 2914 or $SSGL_PORT if set */
+	done_with_lobby = 0; //without this line 952 will be set wrong if we have connected to a server once already
```

Comment:
Thanks, I've been meaning to track this down.

```
$ stg show | ./checkpatch.pl -
ERROR: do not use C99 // comments
#17: FILE: snis_client.c:20706:
+	done_with_lobby = 0; //without this line 952 will be set wrong if we have connected to a server once alread

total: 1 errors, 0 warnings, 7 lines checked

Your patch has style problems, please review.

If any of these errors are false positives, please ignore them
or fix the script.
```

Also, don't refer to line numbers in comments, they will swiftly become wrong with time.